simplified-bookmarks-api/src/main/java/org/nypl/simplified/bookmarks/api/BookmarkAnnotationsJSON.kt

```diff
@@ -383,7 +383,8 @@ object BookmarkAnnotationsJSON {
     val objectNode = objectMapper.createObjectNode()
     objectNode.put("@type", "LocatorAudioBookTime")
     objectNode.put("chapter", bookmark.location.chapter)
-    objectNode.put("time", bookmark.location.offsetMilliseconds)
+    objectNode.put("startOffset", bookmark.location.startOffset)
+    objectNode.put("time", bookmark.location.currentOffset)
```

Comment:
When serializing, `time` is set here to be `currentOffset`. But this doesn't seem to be what is expected by the deserialization code below. Or am I missing something?